### ISSUE TITLE
[FEATURE] DownScaffold

### ISSUE THREAD

**Issue by Doogie13** [John200410/rusherhack-issues#809]
**What feature would you like to be added.**
While holding sprint will scaffold down

**Is your feature request related to a problem? Please describe.**
While scaffolding in pvp its more beneficial to be on the low ground means going down is useful to avoid damage and cause your opponent to be on higher ground in a bridge fight

**Additional context**
Lots of 1.8 clients have this

**Comment by FluorideHistorian** [comment]
i dont think this would be possible on any modern anticheat and i have never seen anyone using scaffold unironically in cpvp
